File: Day 2/problem1.c
// https://www.hackerearth.com/practice/basic-programming/input-output/basics-of-input-output/practice-problems/algorithm/mojtaba-prepares-contest-29b2a044/

#include <stdio.h>
#include <string.h>

int main() {
	char *s;
    s = malloc(1024 * sizeof(char));
    scanf("%[^\n]", s);
    s = realloc(s, strlen(s) + 1);
    int i,x=0,y=0;
    for(i = 0; i < strlen(s); i++){
        if(s[i] == 'z') x++;
        else if(s[i] == 'o') y++;
    }
	if(2*x == y) printf("Yes");
	else printf("No");
	return 0;
}

File: Day 3/_problem1.py
# https://www.hackerearth.com/practice/basic-programming/bit-manipulation/basics-of-bit-manipulation/practice-problems/algorithm/navi-and-maths/

# Finally found a solution

for t in range(int(input())):
    input()
    P = [(1, 0, 0)]
    mod = int(1e9 + 7)
    for xarr in map(int,input().split()):
        for j in range(len(P)):
            x,y,z = P[j]
            P.append(( (x*xarr)%mod, (y+xarr)%mod, z+1 ))
    print("Case #%d:" % (t+1), max((x*pow(y, mod-2, mod))%mod for x, y, z in P if z>1))